### ISSUE TITLE
So, i wanted to ear a new scary story to get in the mood of the day... and this happen:

### ISSUE THREAD

**Issue by MonikaEndDie** [Monika-After-Story/MonikaModDev#8121]
[code]
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script-ch30.rpy", line 1510, in script call
  File "game/event-handler.rpy", line 2530, in script call
  File "game/script-stories.rpy", line 136, in script call
  File "game/script-stories.rpy", line 236, in script
  File "game/script-stories.rpy", line 238, in <module>
  File "game/script-stories.rpy", line 116, in get_and_unlock_random_story
IndexError: list index out of range

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script-ch30.rpy", line 1510, in script call
  File "game/event-handler.rpy", line 2530, in script call
  File "game/script-stories.rpy", line 136, in script call
  File "game/script-stories.rpy", line 236, in script
  File "C:\Doki Doki Literature Club Monika after story\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Doki Doki Literature Club Monika after story\renpy\python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script-stories.rpy", line 238, in <module>
  File "game/script-stories.rpy", line 116, in get_and_unlock_random_story
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/random.py", line 275, in choice
IndexError: list index out of range

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.4
[/code]

**Comment by PavelPlayRo** [comment]
yea i get the same error

**Comment by Booplicate** [comment]
#8116

**Comment by Iceninja64** [comment]
I'm also having the same problem when I try to ask for a new story but the good news is that you can reread some of the other stories if you have them unlocked

**Comment by ThePotatoGuy** [comment]
[closing as fixed]
